backend/fms_core/templates.py

```diff
@@ -17,6 +17,7 @@
     "LIBRARY_PREPARATION_TEMPLATE",
     "LIBRARY_QC_TEMPLATE",
     "NORMALIZATION_TEMPLATE",
+    "NORMALIZATION_PLANNING_TEMPLATE",
```

Comment:
Looks like I'll have to update the template prefiller branch after this gets merged ;(